entity.py

```diff
@@ -9,3 +9,4 @@ class Entity:
     def __init__(self, sprite_name: str, x: int, y: int):
         self.sprite = Sprite(sprite_name)
         self.pos = Vector2(x, y)
+        print("Sprite has been created with the name: " + sprite_name)
```

Comment:
this is good, later we should probably make some `if` check for if we want debug logs like you have with the position logging in main

Reply:
true true, can we set it up to default as off for this specifically?

Reply:
Leave it for now I'll make an issue for it